website/content/en/status/report-2023-07-2023-09/desktop-convenience.adoc

```diff
@@ -0,0 +1,26 @@
+=== Filling gaps in the FreeBSD desktop experience
+
+Links: +
+link:https://github.com/outpaddling/qmediamanager[External media manager] URL: link:https://github.com/outpaddling/qmediamanager[] +
+link:https://github.com/outpaddling/devd-mount[External media manager] URL: link:https://github.com/outpaddling/devd-mount[] +
+link:https://github.com/outpaddling/npmount[External media manager] URL: link:https://github.com/outpaddling/npmount[] +
+link:https://github.com/outpaddling/freebsd-update-notify[External media manager] URL: link:https://github.com/outpaddling/freebsd-update-notify[] +
```

Comment:
I believe only the first of those links was intended to be rendered as "External media manager".